### PR TITLE
Fix link the actor lifecycle

### PR DESCRIPTION
**Pull request by vitornp**

### CODE REVIEW

File: akka-docs/src/main/paradox/scala/guide/tutorial_1.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ We see that when we stopped actor `first` it recursively stopped actor `second`
 This ordering is strict, _all_ `postStop()` hooks of the children are called before the `postStop()` hook of the parent
 is called.
 
-The family of these lifecycle hooks is rich, and we recommend reading [the actor lifecycle](http://doc.akka.io/docs/akka/current/scala/actors.html#Actor_Lifecycle) section of the reference for all details.
+The family of these lifecycle hooks is rich, and we recommend reading [the actor lifecycle](http://doc.akka.io/docs/akka/current/scala/actors.html#actor-lifecycle) section of the reference for all details.
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/guide/tutorial_1.md]: Hmm, I guess we should really make this a `@ref[]` link so in the future [paradox can check it](https://github.com/lightbend/paradox/issues/120), could you make that change?

**Review by raboof — CHANGES_REQUESTED** [review]
Thanks for noticing and taking the time to propose this improvement! I think we can do even better, could you consider using `@ref` here?

**Comment by vitornp** [comment]
@raboof Change for `@ref`

File: akka-docs/src/main/paradox/scala/guide/tutorial_1.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ We see that when we stopped actor `first` it recursively stopped actor `second`
 This ordering is strict, _all_ `postStop()` hooks of the children are called before the `postStop()` hook of the parent
 is called.
 
-The family of these lifecycle hooks is rich, and we recommend reading [the actor lifecycle](http://doc.akka.io/docs/akka/current/scala/actors.html#Actor_Lifecycle) section of the reference for all details.
+The family of these lifecycle hooks is rich, and we recommend reading @ref:[the actor lifecycle](actors.html#actor-lifecycle) section of the reference for all details.
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/guide/tutorial_1.md]: I'm afraid the syntax is `@ref[the actor lifecycle](../actors.md#actor-lifecycle)` - you can run 'sbt paradox' to test but I appreciate that's not so easy from the GitHub UI :)

**Review by raboof — CHANGES_REQUESTED** [review]

**Comment by raboof** [comment]
PLS BUILD

**Comment by vitornp** [comment]
PLS BUILD

**Review by raboof — APPROVED** [review]
Thanks, LGTM when Jenkins agrees :)

**Comment by vitornp** [comment]
@raboof Jenkins automatic start?

**Comment by raboof** [comment]
Hmm, it should but apparently didn't

**Comment by raboof** [comment]
PLS BUILD

**Review by johanandren — APPROVED** [review]

**Pull request merged**